setup.cfg

```diff
@@ -0,0 +1,42 @@
+[metadata]
+name = pytest-astropy
+url = https://astropy.org
+author = The Astropy Developers
+author_email = astropy.team@gmail.com
+classifiers =
+    Development Status :: 3 - Alpha
+    Framework :: Pytest
+    Framework :: Hypothesis
+    Intended Audience :: Developers
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: Implementation :: CPython
+    Topic :: Software Development :: Testing
+    Topic :: Utilities
+license = BSD
+description = Meta-package containing dependencies for testing
+long_description = file: README.rst
+keywords = pytest, py.test, remotedata, openfiles, doctestplus
```

Comment:
Pytest maintainer :tophat::  the `py.test` alias is discouraged.

Hypothesis :tophat:: could add `hypothesis`, `property-based testing`

Reply:
Fixed!